be/src/http/action/transaction_stream_load.cpp

```diff
@@ -410,23 +419,53 @@ void TransactionStreamLoadAction::on_chunk_data(HttpRequest* req) {
     auto evbuf = evhttp_request_get_input_buffer(ev_req);
 
     int64_t start_read_data_time = MonotonicNanos();
-    while (evbuffer_get_length(evbuf) > 0) {
-        ByteBufferPtr bb = ByteBuffer::allocate(4096);
+
+    size_t len = 0;
+    while ((len = evbuffer_get_length(evbuf)) > 0) {
```

Comment:
It's better to add a test case to test the large json.

Reply:
I'll add them to the daily test.